package.json

```diff
@@ -80,6 +81,58 @@
         }
     },
     "contributes": {
+        "walkthroughs": [
+            {
+                "id": "jupyterWelcome",
+                "title": "Get started with Jupyter Notebooks",
+                "description": "Your first steps to set up a Jupyter project with all the powerful tools and features that the Jupyter extension has to offer!",
+                "when": "false",
```

Comment:
when: false? Does this mean it doesn't show up?

Reply:
Yep, by default it's turned off. The experiment turns it on for folks:

vscode.gettingStarted.overrideCategory.ms-toolsai.jupyter.jupyterWelcome.when:workspacePlatform != webworker